jinja/jinjify.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+"""This script takes two parameters.  The first is a latex file with
+yaml frontmatter.  The body after the frontmatter is latex use as a
+jinja2 template.  We pass the yaml frontmatter to that jinja2 template
+and print the results.
+
+"""
+import os
+
+# Search path for Jinja templates, in order of preference
+TEMPLATE_DIRS = [ os.path.abspath(os.getcwd()),
+                  os.path.join(os.getenv('OTSDIR'), 'forms', 'jinja'),
+                  os.path.join(os.getenv('OTSDIR'), 'forms', 'latex'),
+                  os.path.join(os.getenv('OTS_DOCTOOLS_DIR'), 'jinja'),
+                  os.path.join(os.getenv('OTS_DOCTOOLS_DIR'), 'latex')]
```

Comment:
The search order here and in [Makefile.docspecific](https://github.com/OpenTechStrategies/ots-doctools/blob/pipeline/Makefile.docspecific#L9) seem to differ.
Additionally, if the environment variable `OTSDIR` is not found, `os.getenv` returns `None` and `os.path.join` will raise an error:
```
TypeError: expected str, bytes or os.PathLike object, not NoneType
``` 
The top level Makefile does not run without `OTS_DOCTOOLS_DIR` being set as documented in the [README](https://github.com/OpenTechStrategies/ots-doctools/blob/pipeline/README.md#L22), so it's probably safe to assume `OTS_DOCTOOLS_DIR` is set when the pipeline is run. As the [README](https://github.com/OpenTechStrategies/ots-doctools/blob/pipeline/README.md) does not mention `OTSDIR`, it seems to be optional.  If that's the case, I'd suggest:
```suggestion
TEMPLATE_DIRS = [ os.path.abspath(os.getcwd()),
                  os.path.join(os.getenv('OTS_DOCTOOLS_DIR'), 'jinja'),
                  os.path.join(os.getenv('OTS_DOCTOOLS_DIR'), 'latex')]
if os.getenv('OTSDIR'):
    TEMPLATE_DIRS += [
                  os.path.join(os.getenv('OTSDIR'), 'forms', 'jinja'),
                  os.path.join(os.getenv('OTSDIR'), 'forms', 'latex')]
```

Reply:
@choeppler, thank you for the review!  Your suggested change makes sense.  I've committed it to `master` in commit c1d21ddb2436.

I'll close this PR, too.  All the commits in it have already been applied to `master`.  There must have been a rebasing somewhere along the way, because they have different commit IDs there than they do here.